x/btcstaking/types/btc_delegation.go

```diff
@@ -198,25 +198,14 @@ func (d *BTCDelegation) IsSignedByCovMember(covPk *bbn.BIP340PubKey) bool {
 // AddCovenantSigs adds signatures on the slashing tx from the given
 // covenant, where each signature is an adaptor signature encrypted by
 // each finality provider's PK this BTC delegation restakes to
-func (d *BTCDelegation) AddCovenantSigs(covPk *bbn.BIP340PubKey, sigs []asig.AdaptorSignature, quorum uint32) error {
-	// we can ignore the covenant sig if quorum is already reached
-	if d.HasCovenantQuorums(quorum) {
-		return nil
-	}
-
-	if d.IsSignedByCovMember(covPk) {
-		return nil
-	}
-
+func (d *BTCDelegation) AddCovenantSigs(covPk *bbn.BIP340PubKey, sigs []asig.AdaptorSignature, quorum uint32) {
```

Comment:
```suggestion
func (d *BTCDelegation) AddCovenantSigs(covPk *bbn.BIP340PubKey, sigs []asig.AdaptorSignature) {
```
Quorum is not verified in this function anymore

Reply:
Also this modification seems to add a contract that `AddCovenantSigs` shall not take duplicated covenant adaptor signatrues. While it might be acceptable given that `BTCDelegation` will be entirely a internal database object, maybe it's a good idea to document these contracts in the docstring. Wdyt?

Reply:
hum I just see those two methods adding signatures as little helper to mutate object state, although I agres that it is good idea to add some doc string that it it up to the caller to do all verifications 👍